api/open-vsx.ts

```diff
@@ -0,0 +1,59 @@
+import millify from 'millify'
+import got from '../libs/got'
+import { version, versionColor } from '../libs/utils'
+import { createBadgenHandler, PathArgs } from '../libs/create-badgen-handler'
+
+export default createBadgenHandler({
+  title: 'Open VSX',
+  examples: {
+    '/open-vsx/v/idleberg/electron-builder': 'version',
+    '/open-vsx/r/idleberg/electron-builder': 'reviews',
+    '/open-vsx/l/idleberg/electron-builder': 'license',
+    '/open-vsx/d/idleberg/electron-builder': 'downloads'
+  },
+  handlers: {
+    '/open-vsx/:topic/:namespace/:pkg': handler
```

Comment:
@amio Is it preferred to use `:topic<v|version|r|reviews|l|license|d|downloads>` or unconstrained `:topic` param in combination with default _unknown topic_ case?

Reply:
Both are fine, if there are lots of options, an unconstrained  `:topic` would be handy, if it's just a few, `:topic<a|few|option>` would be prefered. While there's no clear line between the two.